src/Connection/Aggregate/RedisCluster.php

```diff
@@ -276,11 +280,11 @@ protected function guessNode($slot)
      */
     protected function createConnection($connectionID)
     {
-        $separator = strrpos($connectionID, ':');
+        list($host,$port) = explode(':', $connectionID, 2);
```

Comment:
if your goal is to make code more readable/understandable we should go with strrpos instead of reversing the IP parts :)